source/s3_client.c

```diff
@@ -314,35 +330,10 @@ void aws_s3_client_release(struct aws_s3_client *client) {
     aws_ref_count_release(&client->ref_count);
 }
 
-void aws_s3_client_push_meta_request(struct aws_s3_client *client, struct aws_s3_meta_request *meta_request) {
+static void s_s3_client_reset_active_synced(struct aws_s3_client *client) {
     AWS_PRECONDITION(client);
-    AWS_PRECONDITION(client->vtable);
```

Comment:
Also not gone, just moved next to their "default" virtual implementation to make their placement in the source file less random.